src/tests/auth/__snapshots__/AuthModal.test.js.snap

```diff
@@ -0,0 +1,472 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
ten plik powinien tu być?

Reply:
tak, bo to snapshot, ale chyba zrezygnuje z snapshotów :)